### DISCUSSION TITLE
Training stops when Monitor goes into sleeping mode.

### DISCUSSION THREAD

**Discussion by RealBeyondMaster** [kohya-ss/sd-scripts#531] [Q&A]
Hi all

As mentioned in the title, the lora training makes a pause when the monitor is asleep, once the monitor goes active again, you can
press enter in the command shell and it resumes the training.

Is there any way to prevent this? I don't want my monitor running most of the time while I am not at the pc and it's doing the training.

Thank you

**Comment by TingTingin** [comment]
this is related to your pc not to lora training if your pc pauses execution when your monitor turns off you need to change that setting
